apps/common-app/src/examples/WorkletRuntimeExample.tsx

```diff
@@ -74,7 +74,7 @@ function CreateWorkletRuntimeDemo() {
     const runtime = createWorkletRuntime('foo');
     console.log(runtime);
     console.log(runtime.name);
-    console.log(`${runtime}`);
+    console.log(`${runtime.name}`);
```

Comment:
This is the same as `console.log` above. The purpose of this `console.log` was to check the output of `${runtime}`. Can we restore the original `console.log`?